<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookCar</class>
 <widget class="QDialog" name="bookCar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="BookcarTitle">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Fax</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Book Car</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-200</x>
     <y>0</y>
     <width>1231</width>
     <height>591</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Users/vikram/OneDrive/Desktop/mahindra-scorpio.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="AgelineEdit">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>140</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:2px solid grey;
color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PickupPointLabel">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>210</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Fax</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Pickup Point:</string>
   </property>
  </widget>
  <widget class="QLabel" name="AgeLabel">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>130</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Fax</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Age:</string>
   </property>
  </widget>
  <widget class="QLabel" name="DestinationLabel">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>250</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Fax</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Destination:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NamelineEdit">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>90</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:2px solid grey;
color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="selectCarLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>310</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Fax</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Select Car:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BookCarbtn">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>510</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>Book Car</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PickupuPointLineEdit">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>220</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:2px solid grey;
color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exitbtn">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>510</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(175, 175, 175);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QListWidget" name="SelectCarlistWidget">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>310</y>
     <width>181</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:2px solid grey;
color:white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="NameLabel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>80</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Fax</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="FemaleradioButton">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>female</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="MaleradioButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
</string>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DestinationLineEdit">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>260</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:2px solid grey;
color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>BookcarTitle</zorder>
  <zorder>AgelineEdit</zorder>
  <zorder>PickupPointLabel</zorder>
  <zorder>AgeLabel</zorder>
  <zorder>DestinationLabel</zorder>
  <zorder>NamelineEdit</zorder>
  <zorder>selectCarLabel</zorder>
  <zorder>BookCarbtn</zorder>
  <zorder>PickupuPointLineEdit</zorder>
  <zorder>exitbtn</zorder>
  <zorder>SelectCarlistWidget</zorder>
  <zorder>NameLabel</zorder>
  <zorder>FemaleradioButton</zorder>
  <zorder>MaleradioButton</zorder>
  <zorder>DestinationLineEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
